### ISSUE TITLE
Do Carriage-Returns [\r] in log files break FilePulse?

### ISSUE THREAD

**Issue by rakefs** [streamthoughts/kafka-connect-file-pulse#344]
Does DOS line endings [\r\n] break FilePulse?
I have a source log file where every line is ended with a carriage return and a newline (0x0D0A).  
What I have found is that the connector works, initially, and is able to parse multiline records up until the end of the file.
The source (multiline) log file will be consumed and messages are committed to Kafka.
However, any subsequently appended records to that initial pass-through of the log file are not recognized.

**Comment by fhussonnois** [comment]
Hi @rakefs, when filepulse reaches the end of a file it will consider it to be processed completely after some time. This could be the reason why no records subsequently added to the file are captured. When using the `RowFileInputReader` you can configure the property: 

*  `read.max.wait.ms`: The maximum time to wait in milliseconds for more bytes after hitting end of file.

see documentation: https://streamthoughts.github.io/kafka-connect-file-pulse/docs/developer-guide/file-readers/#rowfileinputreader-default

**Comment by rakefs** [comment]
I am still trying to verify the "read.max.wait.ms" as one source of the problem with trying to maintain an active listener on a live log file. I have started a new issue to try to track down the overall problem I am having with reading a log file with multiline records: [#389 ](https://github.com/streamthoughts/kafka-connect-file-pulse/issues/389)

The answer to the original question in this issue about DOS line endings seems to be that they are not a problem. I am using the following and it seems to work as expected:
```
"filters.MRF.separator": "\\r\\n",
```
Closing this issue/question.
